site/data/scans/ssti/all.yml

```diff
@@ -178,12 +178,12 @@ details:
   any: Pass
 - title: 'Input rendered in other location'
   rule_40012: FAIL
-  rule_40026: Pass
+  rule_40026: FAIL
```

Comment:
This seems to fail consistently now (or more often).

Reply:
Worth merging this one then?